socorro/cron/jobs/matviews.py

```diff
@@ -90,48 +80,6 @@ class BuildADUCronApp(_MatViewBackfillBase):
     depends_on = ('fetch-adi-from-hive', 'reports-clean')
 
 
-class CrashesByUserCronApp(_MatViewBackfillBase):
-    proc_name = 'update_crashes_by_user'
-    app_name = 'crashes-by-user-matview'
```

Comment:
We sorted out in the meeting that we can remove these. We only care about the historical data that is in the table.